pyqtgraph/graphicsItems/TextItem.py

```diff
@@ -59,12 +60,12 @@ def __init__(self, text='', color=(200,200,200), html=None, anchor=(0,0),
 
     def setText(self, text, color=(200,200,200)):
         """
-        Set the text and color of this item. 
+        Set the text of this item. 
+        
+        The color entry is deprecated and kept to avoid an API change.
```

Comment:
But this *is* an API change because now `setText` silently ignores the `color` kwarg.  Perhaps a better solution would be to make the default color `None` and then 

```python
if color is not None:
    self.setColor(color)
```

Doing it this way will not break existing code that *is* calling `setText` with a color kwarg, but will get the desired behavior of not having to send in the current text color every time you want to change the text.